src/platform/interpreter/workspaceInterpreterTracker.ts

```diff
@@ -46,6 +47,9 @@ export class WorkspaceInterpreterTracker implements IExtensionSyncActivationServ
         return areInterpreterPathsSame(activeInterpreterPath, interpreter.uri);
     }
     private trackActiveInterpreters() {
+        if (this.webExtension) {
+            return;
+        }
```

Comment:
To be fixed properly here https://github.com/microsoft/vscode-jupyter/issues/13610